client/my-sites/earn/components/form-text-input-with-value-generation/index.tsx

```diff
@@ -32,6 +35,8 @@ const FormTextInputWithValueGeneration = ( {
 	disabled = false,
 	isError = false,
 	isValid = false,
+	textInputAriaLabel = translate( 'enter value' ),
```

Comment:
We usually use sentence case for such strings and use CSS if we need to enforce lowercasing.